translations/mod-fqm-manager/en.json

```diff
@@ -224,11 +224,11 @@
   "entityType.drv_purchase_order_line_details.purchase_order_line_updated_by": "Purchase order line updated by",
   "entityType.drv_purchase_order_line_details.purchase_order_line_updater_id": "Purchase order line updated by ID",
   "entityType.drv_purchase_order_line_details.purchase_order_line_updated_date": "Purchase order line updated date",
-  "entityType.drv_purchase_order_line_details.purchase_order_notes": "PO notes",
-  "entityType.drv_purchase_order_line_details.purchase_order_type": "PO type",
-  "entityType.drv_purchase_order_line_details.purchase_order_updated_by": "PO updated by",
-  "entityType.drv_purchase_order_line_details.purchase_order_updated_by_id": "PO updated by ID",
-  "entityType.drv_purchase_order_line_details.purchase_order_updated_date": "PO updated date",
+  "entityType.drv_purchase_order_line_details.po_notes": "PO notes",
```

Comment:
These entries should be updated in all the English translation files. (Technically, things will probably work even if we don't, but it doesn't make sense to have entries for BE columns like purchase_order_notes in the other files, since we won't have columns with those names anymore)